src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,439 @@
+choose to show the permission prompt dynamically upon the first attempt to use any
+of the methods of the new API.
+Read on to learn more.
+
+### The `isMultiScreen()` method
+
+The first improvement of the Multi-Screen Window Placement API is the introduction of the
+`Window.isMultiScreen()` method. It returns a promise that resolves to either `true` or `false`,
+depending on whether one or multiple screens are currently connected to the machine. For my setup,
+it returns `true`.
+
+```js
+await window.isMultiScreen();
+// Returns `true` or `false`.
+```
+
+### The `getScreens()` method
+
+Now that I know that the current setup with my iPad is a multi-screen one, I can obtain more
+information about the second screen via the new `Window.getScreens()` method. It returns a promise
+that resolves with an array of `Screen` objects. On my MacBook Air 13 with a connected iPad, this
+returns an array of two `Screen` objects:
+
+```js
+await window.getScreens();
+/* Output from my MacBook Air 13″ with the iPad attached:
+  Screen 1 (built-in display):
+  availHeight: 975
+  availLeft: 0
+  availTop: 23
+  availWidth: 1680
+  colorDepth: 30
+  height: 1050
+  id: "0"
+  internal: true
+  left: 0
+  orientation: null
+  pixelDepth: 30
+  primary: true
+  scaleFactor: 2
+  top: 0
+  touchSupport: false
+  width: 1680
+
+  Screen 2 (iPad):
+  availHeight: 1001
+  availLeft: 1680
+  availTop: 23
+  availWidth: 1366
+  colorDepth: 24
+  height: 1024
+  id: "1"
+  internal: false
+  left: 1680
+  orientation: null
+  pixelDepth: 24
+  primary: false
+  scaleFactor: 2
+  top: 0
+  touchSupport: false
+  width: 1366
+*/
+```
+
+Note how the value of `left` for the iPad starts at `1680`, which is exactly the `width` of the
+built-in display. This allows me to determine exactly how the screens are arranged logically (next
+to each other, on top of each other, etc.). There is also data now for each screen to show whether
+it is an `internal` one and whether it is a `primary` one. Both also have an `id`, which, if
+persisted across browser sessions, allows for window arrangements to be restored exactly.
+
+### The `onscreenschange` event
+
+The only thing missing now is a way to detect when my screen setup changes. A new event
+`onscreenschange` (plural "screens") does exactly that: it fires whenever the screen constellation
+is modified. This can happen when the resolution of one of the connected screens changes or when a
+new or an existing screen is (physically or virtually in the case of Sidecar) plugged or unplugged.
+
+```js
+console.log("Before:", await window.getScreens());
+window.addEventListener("screenschange", async () => {
+  console.log("After:", await window.getScreens());
+});
+```
+
+Note that you need to look up the new screen details asynchronously, the `screenschange` event
+itself does not provide this data
+(see
+[Issue&nbsp;28](https://github.com/webscreens/window-placement/issues/28)
+for details).
```

Comment:
```suggestion
itself does not provide this data. [This may change in the
future](https://github.com/webscreens/window-placement/issues/28). For now you
can look up the screen details by calling `window.getScreens()` as shown below.

```js
window.addEventListener('screenschange', async (e) => {
  console.log('I am there, but mostly useless', e);
  const details = await window.getScreens();
});
```

```

Reply:
I didn't leave off the closing back ticks. GitHub is swallowing them. You'll need to add them to the source.